README.md

```diff
@@ -102,7 +101,8 @@ Finally, build and start vector itself:
         + 1013 hidden modules
     ```
    Remember, the command will not terminate since it runs the web server
-   and rebuilds source files when they change.
+   and rebuilds source files when they change. This development server also
+   disables caching, so do NOT use it in production.
 1. Open http://127.0.0.1:8080/ in your browser to see your newly built Vector.
 
 When you make changes to `matrix-js-sdk` or `matrix-react-sdk`, you will need
```

Comment:
unrelated, but this looks like a lie: `matrix-js-sdk` doesn't have a build step so should be removed from this paragraph (and wording tweaked thereafter)